setup.py

```diff
@@ -48,7 +48,7 @@
 EXTRAS_REQUIRE = {
     'optional': ['cython', 'ephem', 'netcdf4', 'nrel-pysam', 'numba',
                  'pvfactors', 'siphon', 'statsmodels',
-                 'cftime >= 1.1.1'],
+                 'cftime >= 1.1.1', 'scikit-image'],
```

Comment:
This merits an entry in the `Requirements` section in the what's new file IMHO

Reply:
Yeah it is kind of a major requirement. Actually, it may be smarter to just remove that, since loading the DEM as an image does not capture all the metadata. This would get rid of the need for skimage.

Reply:
Is there an alternative representation of the DEM data that would be more useful when calculating horizon profiles?  Might be worth splitting all the DEM stuff into a separate PR that does both DEM retrieval and manipulation.

Reply:
The problem with the DEMs is that there's not really a standard format; .hgt & .tiff seem to be the most common, but to handle them properly is best done with a dedicated library like GDAL. However, GDAL is in C and requires a pretty complicated install process that I would hesitate to add to pvlib, even optionally. The actual algorithm for calculating the horizon is overshadowed by the necessary wrapper code to handle everything properly. We've been going back and forth on if that part would be usable in pvlib, or as a separate package, since it ends up as more of a geography package then anything related to PV.

My other concern with that is that the vast majority of the time, I would advise somebody interested in modeling PV performance to just use the pvgis method added here. The only reason I'd foresee actually needing to download the DEM is if you were also modeling uneven terrain, and since you'd already have the DEM you might as well calculate the horizon as well. But all in all, I think adding code to process DEMs to pvlib might not be worth the feature creep/support cost when we can just point people to pvgis anyway. 

Anyway, I think in any case, you're right, the best thing to do is to split it from this PR and we can re-evaluate including it later?